.changeset/stale-chefs-retire.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/core': minor
```

Comment:
This'll be a `patch` change as it's not breaking an we're still on 0.x

Reply:
Thanks, let me update the changeset